mne/time_frequency/tfr.py

```diff
@@ -927,6 +928,52 @@ def apply_baseline(self, baseline, mode='mean', verbose=None):
             If not None, override default verbose level (see mne.verbose).
         """
         self.data = rescale(self.data, self.times, baseline, mode, copy=False)
+        return self
+
+    def plot_gfp(self, baseline=None, mode='mean', colors='auto',
+                 summarize=False, title=None, show=True):
+        """ Show time and frequency resolved global field power
+
+        Parameters
+        ----------
+        tfr : instance of AverageTFR
+            The MNE time frequency object.
+        baseline : None (default) or tuple of length 2
+            The time interval to apply baseline correction.
+            If None do not apply it. If baseline is (a, b)
+            the interval is between "a (s)" and "b (s)".
+            If a is None the beginning of the data is used
+            and if b is None then b is set to the end of the interval.
+            If baseline is equal ot (None, None) all the time
+            interval is used.
+        mode : None | 'logratio' | 'ratio' | 'zscore' | 'mean' | 'percent'
+            Do baseline correction with ratio (power is divided by mean
+            power during baseline) or zscore (power is divided by standard
```

Comment:
all this is copied from the other doc strings, see tfr module

On Wed, Apr 13, 2016 at 9:27 PM Jean-Rémi KING <notifications@github.com>
wrote:

> In mne/time_frequency/tfr.py
> <https://github.com/mne-tools/mne-python/pull/3123#discussion_r59619135>:
>
> > +
> > +        Parameters
> > +        ----------
> > +        tfr : instance of AverageTFR
> > +            The MNE time frequency object.
> > +        baseline : None (default) or tuple of length 2
> > +            The time interval to apply baseline correction.
> > +            If None do not apply it. If baseline is (a, b)
> > +            the interval is between "a (s)" and "b (s)".
> > +            If a is None the beginning of the data is used
> > +            and if b is None then b is set to the end of the interval.
> > +            If baseline is equal ot (None, None) all the time
> > +            interval is used.
> > +        mode : None | 'logratio' | 'ratio' | 'zscore' | 'mean' | 'percent'
> > +            Do baseline correction with ratio (power is divided by mean
> > +            power during baseline) or zscore (power is divided by standard
>
> z-score
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/3123/files/8f3d7330b6639853e22ab97e4a2fc36bfa281cef#r59619135>
>